x-pack/plugins/uptime/public/components/monitor/synthetics/waterfall/components/constants.ts

```diff
@@ -16,3 +16,5 @@ export const FIXED_AXIS_HEIGHT = 32;
 
 // number of items to display in canvas, since canvas can only have limited size
 export const CANVAS_MAX_ITEMS = 150;
+
+export const CHART_LEGEND_PADDING = 62;
```

Comment:
why specifically 62? Just wondering.

Reply:
this should be potentially improved to get height from legend ref.